stackrox/stackrox
# Remove ROX_DECOMMISSIONED_CLUSTER_RETENTION feature flag in ui code

## Description

See https://github.com/stackrox/stackrox/tree/master/ui/apps/platform#delete-a-feature-flag-from-frontend-code

Steps 1, 2, 3.

**Charmik**, can you do step 4 when you remove occurrences from backend?

Delete `ci_export ROX_WHATEVER "${ROX_WHATEVER:-true}"` from `export_test_environment` function in tests/e2e/lib.sh

## Checklist
- [x] Deleted conditional skips from integration tests

## Testing Performed

1. `yarn build` in ui
2. `yarn lint` in ui
3. `yarn cypress-open` in ui/apps/platform
    * clusters/clusterDeletion.test.js (edited)
    * clusters/clusters.test.js
    * clusters/clustersCertificateExpiration.test.js
    * clusters/clustersHealthStatus.test.js (edited)

### manual testing

1. Visit /main/clusters and see **Cluster Deletion** column

2. Click cluster row and scroll down to see **Cluster Deletion** card

3. Visit /main/systemconfig and scroll down to see **Cluster Deletion** heading
